app/controllers/admin/products_controller.rb

```diff
@@ -1,6 +1,15 @@
 module Admin
   class ProductsController < AdminController
 
+    def index
+      @source_models = Product.where(state: Product::ACTIVE)
+    end
+
+    def destroy
```

Comment:
this is the same as in coupons controller, create a rails concern for this so you don't have the logic in 2 places